js/huobi.js

```diff
@@ -2570,10 +2574,8 @@ module.exports = class huobi extends Exchange {
                 const code = this.safeCurrencyCode (currencyId);
                 result[code] = account;
             } else if (isolated) {
-                const fieldName = future ? 'futures_contract_detail' : 'contract_detail';
```

Comment:
These fields are not isolated margin related but cross. In this case, I don't really find them useful so they're not being used even in cross balance parsing
![image](https://user-images.githubusercontent.com/43336371/162422241-2e571b3f-3830-4c2d-af1c-20ad69659692.png)